### ISSUE TITLE
District + Garden page review

### ISSUE THREAD

**Issue by morganfoust** [bluecadet/longwood-d8#400]
- [ ] Garden page carousel. No content appears on the 9 of 9 image, and the image is the same as the first image (it just shifts slightly when you click next).
![ezgif com-gif-maker (15)](https://user-images.githubusercontent.com/50842575/157477133-15fa12e9-7900-4f0a-9e0b-964e1edc3464.gif)

- [ ] Garden page WYSIWYG. Minor type tweak: Can we make "About this Garden" lowercase and use the regular weight for "Flower Garden Walk." Before and after pics attached.
<img width="1072" alt="Screen Shot 2022-03-09 at 10 38 36 AM" src="https://user-images.githubusercontent.com/50842575/157477173-7664d32f-a0de-4a5b-822a-442641cf304b.png">

- [ ] Question: Are these photo credits in the bottom right of images? Separate from the captions? I think it would be useful to add in a sample caption here to see the styling and to see how it works with a photo credit. Not sure the right aligned will feel right.
<img width="344" alt="Screen Shot 2022-03-09 at 10 39 55 AM" src="https://user-images.githubusercontent.com/50842575/157477267-b4df7fef-5326-41db-adb7-72e5866f0466.png">

- [ ] Question: Is there a reason there are two separate ticket links? If not, let's remove that additional CTA.
<img width="344" alt="Screen Shot 2022-03-09 at 10 39 55 AM" src="https://user-images.githubusercontent.com/50842575/157477935-3aa55be9-4166-458b-8a88-f684020c63f7.png">

- [ ] Let's remove the underline from Map until you hover over it.
<img width="412" alt="Screen Shot 2022-03-09 at 10 51 42 AM" src="https://user-images.githubusercontent.com/50842575/157478103-b58bd151-2dfe-4f03-b877-682a83432735.png">

- [ ] When you hover over any item in the top navigation, they all become underline. Only the one you are hovering on should be underlined here:
![ezgif com-gif-maker (15)](https://user-images.githubusercontent.com/50842575/157478372-050c9e7e-1a4f-49c9-b0f8-d8fa861a46b0.gif)

- [ ] For alerts with white text, can we have the explanation point match the fill color of the alert? In this case, red.
<img width="255" alt="Screen Shot 2022-03-09 at 10 55 09 AM" src="https://user-images.githubusercontent.com/50842575/157478884-fee1fdc0-e26f-499e-bac8-9e1d884b0f92.png">
 

- [ ] Question: Is there only one garden in the House and Theater district? I have a feeling they won't respond well to the white space there.

- [ ] Event callout. The title should say "Events" or something similar, and the term "event" should not be showing up in the title of the event itself. The event itself should use the regular weight, not bold font. The "explore all events" styling is incorrect and should use a button with a fill. Lastly, this differs from the Figma - but seeing it live, I think we should make the date and time information the lighter weight instead of bold.
<img width="1276" alt="Screen Shot 2022-03-09 at 11 00 15 AM" src="https://user-images.githubusercontent.com/50842575/157479941-38f8e92f-1362-4811-8d3a-9e6eec61b17f.png">

- [ ] There's a lot of white space on the right hand side of the pages on mobile that you can pan to:
![ezgif com-gif-maker (15)](https://user-images.githubusercontent.com/50842575/157481146-cbf1f71e-419c-41cf-b1ff-0ae47dac56a2.gif)

This might be the reason? 

- [ ] Testing on mobile (iPhone SE 2020, Safari), the alerts are appearing very large and missing the exclamation mark: ![View recent photos](https://user-images.githubusercontent.com/50842575/157481739-a05dd8a0-0e90-4e98-bc4d-245e4e424c87.jpeg)

- [ ] Mobile carousel: Let's add some padding in between the arrows and images for mobile. This is with 12px of padding:
<img width="391" alt="Screen Shot 2022-03-09 at 11 14 16 AM" src="https://user-images.githubusercontent.com/50842575/157482941-9e8ba18f-71d1-4c67-97ec-5b59fae46604.png">

- [ ] Mobile: I'm able to scroll out of the hamburger menu. Can we update so you can't start scrolling again until you X out? (I'm seeing this on my actual phone, too - iPhone SE 2020, Safari)

- [ ] Is the latin name here... real?
<img width="1041" alt="Screen Shot 2022-03-09 at 11 18 58 AM" src="https://user-images.githubusercontent.com/50842575/157483792-83adc121-cc8a-4f05-b81a-2acf7953c672.png">

**Comment by maxfenton** [comment]
@morganfoust Can you show me where you mean by `Question: Is there a reason there are two separate ticket links? If not, let's remove that additional CTA.` please? Thanks.

**Comment by maxfenton** [comment]
90% done & will re-check once this posts to the live server 
The ones that aren't done have been converted to separate tickets
